src/validation/rules/KnownArgumentNames.js

```diff
@@ -17,22 +17,43 @@ import {
   DIRECTIVE
 } from '../../language/kinds';
 import type { GraphQLType } from '../../type/definition';
-
+import { suggestionList } from '../../utilities/suggestionList';
 
 export function unknownArgMessage(
   argName: string,
   fieldName: string,
-  type: GraphQLType
+  type: GraphQLType,
+  suggestedArgs: Array<string>
 ): string {
-  return `Unknown argument "${argName}" on field "${fieldName}" of ` +
+  let message = `Unknown argument "${argName}" on field "${fieldName}" of ` +
     `type "${type}".`;
+  if (suggestedArgs.length) {
+    const suggestions = suggestedArgs
+      .map(t => `"${t}"`)
+      .join(', ');
+    message += ` Perhaps you meant ${suggestions}?`;
+  } else {
+    message += ' There is no known argument for this field.';
```

Comment:
This seems redundant with the rest of the message. I think if there are no suggested args, you can just leave it at that and not amend to the message.